# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux26/eth-naming.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur linux-2.6.16/drivers/net/wireless/airo.c linux-2.6.16/drivers/net/wireless-unified/airo.c
--- linux-2.6.16/drivers/net/wireless/airo.c	2006-03-20 06:53:29.000000000 +0100
+++ linux-2.6.16/drivers/net/wireless-unified/airo.c	2006-06-06 17:25:52.000000000 +0200
@@ -2659,7 +2659,7 @@
 					struct net_device *ethdev)
 {
 	int err;
-	struct net_device *dev = alloc_netdev(0, "wifi%d", wifi_setup);
+	struct net_device *dev = alloc_netdev(0, "eth%d", wifi_setup);
 	if (!dev)
 		return NULL;
 	dev->priv = ethdev->priv;
diff -ur linux-2.6.16/drivers/net/wireless/hostap/hostap_hw.c linux-2.6.16/drivers/net/wireless-unified/hostap/hostap_hw.c
--- linux-2.6.16/drivers/net/wireless/hostap/hostap_hw.c	2006-03-20 06:53:29.000000000 +0100
+++ linux-2.6.16/drivers/net/wireless-unified/hostap/hostap_hw.c	2006-06-06 17:26:58.000000000 +0200
@@ -82,10 +82,10 @@
 module_param_array(dtim_period, int, NULL, 0444);
 MODULE_PARM_DESC(dtim_period, "DTIM period");
 
-static char dev_template[16] = "wlan%d";
+static char dev_template[16] = "eth%d";
 module_param_string(dev_template, dev_template, sizeof(dev_template), 0444);
 MODULE_PARM_DESC(dev_template, "Prefix for network device name (default: "
-		 "wlan%d)");
+		 "eth%d)");
 
 #ifdef final_version
 #define EXTRA_EVENTS_WTERR 0
